frontend/src/container/FormAlertChannels/index.tsx

```diff
@@ -109,7 +119,9 @@ function FormAlertChannels({
 interface FormAlertChannelsProps {
 	formInstance: FormInstance;
 	type: ChannelType;
-	setSelectedConfig: React.Dispatch<React.SetStateAction<Partial<SlackChannel>>>;
+	setSelectedConfig: React.Dispatch<
+		React.SetStateAction<Partial<SlackChannel & WebhookChannel & PagerChannel>>
```

Comment:
Can we Make this one as one type

Reply:
Good idea. Can do. But this will require complete re-testing. Can we do this in next release.